src/content/1.0.0/library/clojure/clojure-overview.md

```diff
@@ -0,0 +1,208 @@
+## Clojure
+
+[![cljdoc badge](https://cljdoc.org/badge/com.fluree/db)](https://cljdoc.org/d/com.fluree/db/CURRENT)
+
+Fluree has a Clojure client.
+
+### A note on terminology
+
+If you are familiar with Datomic, Datahike, Datascript, or Crux (or other Clojure triple-store+ databases) you may wonder at the difference in terminology between those databases and Fluree. Fluree uses [RDF](https://en.wikipedia.org/wiki/Resource_Description_Framework) terminology to describe the relationships between entities, or subjects.
+
+Here's a quick glossary of how your knowledge may map on to Fluree's terminology:
+
+Clojure Triple-Store+ term | Fluree term
+-- | --
+entity | subject
+attribute | predicate
+value | object
+datom | flake
+
+### Setup
+Make sure you have a running Fluree ledger service to connect to. See the [Getting Started](/docs/1.0.0/getting-started/installation) docs for more details.
+
+Add to the artifact to your dependencies:
+
+Leiningen -
+
+```all
+[com.fluree/db "1.0.0-rc22"]
+```
+
+deps.edn -
+
+```all
+com.fluree/db {:mvn/version "1.0.0-rc22"}
+```
+
+Require the api and establish a connection to your Fluree server:
+
+```all
+(require '[fluree.db.api :as fdb])
+
+(def conn (fdb/connect "http://localhost:8090"))
+
+(def ledger "example/test")
+
+@(fdb/new-ledger conn ledger)
+```
+
+### Transacting
+
+`transact` is used to make any changes to the ledger, either adding data or schema. It's all data, and it all uses the same api to effect changes.
+
+#### Schema
+
+All data transacted into a ledger must conform to a schema. There are some built-in schemas that every ledger starts with that form very useful building blocks for constructing your own. See the [Schema](/docs/1.0.0/schema/overview) documentation for more details.
```

Comment:
In the current version there is not built-in schemas. That is coming with the feature/prefix branch.

I'd suggest rewording this to something like:
All data transacted into a ledger must conform to a schema. We'll transact a basic schema below. See the [Schema] documentation for more details.